smsdk/client.py

```diff
@@ -225,3 +238,102 @@ def get_machine_schema(self, machine_source, types=[], return_mtype=False, **kwa
                             f"Unknow stat schema identified :: machine_type {machine_source} - "
                             f"title_prefix :: {stat.get('display', {}).get('title_prefix', '')}")
         return fields
+
+    def _get_factories(self, normalize=True, *args, **kwargs):
```

Comment:
Can be sepecifed as:
```suggestion
    def _get_factories(self, *args, normalize=True, **kwargs):
```

Reply:
included suggestion